scripts/globals/mobskills/Bombilation.lua

```diff
@@ -0,0 +1,28 @@
+---------------------------------------------
+--  Bombilation
+--
+--  Description: Resets TP of all targets in an area of effect.
+--  Type: Enfeebling
+--  Ignores Shadows
+--  Range: Unknown radial
+---------------------------------------------
+require("scripts/globals/settings");
+require("scripts/globals/status");
+require("scripts/globals/monstertpmoves");
+
+---------------------------------------------
+function onMobSkillCheck(target,mob,skill)
+
+    return 0;
+end;
+
+function onMobWeaponSkill(target, mob, skill)
+   local reset = 0;
```

Comment:
Ok, I made all of these fixes and committed on my end.  If I combine them with the last commit and push them up, will it make the changes for me on this pull request?  Still trying to figure all this process out.  Wren has been super helpful in getting me set up to do it the right way.